src/styles/components/_search.scss

```diff
@@ -1,103 +1,99 @@
-.bsds-search
-{
+.bsds-search {
   width: 100%;
 
-  .bsds-input-text-input
-  {
-    margin-top: 0;
+  .bsds-input-text-input {
     height: 100%;
+    margin-top: 0;
   }
 }
 
-.bsds-search-control
-{
+.bsds-search-control {
   display: flex;
   flex-direction: column;
   gap: space(2x);
 }
 
-.bsds-input-search
-{
+.bsds-input-search {
   position: relative;
   flex: 1;
 }
 
-.bsds-search-clear
-{
+.bsds-search-clear {
   position: absolute;
   right: 0;
+  height: 100%;
   border: 0;
   padding: space(2x);
-  height: 100%;
-  aspect-ratio: 1 / 1;
   background: transparent;
+  aspect-ratio: 1 / 1;
```

Comment:
do we want to look at this re: property order too? i have a feeling anything unspecified may be getting tacked on at the bottom